### PR TITLE
HDDS-6083: EC: Fix flakyness of tests around nodefailures

### PR DESCRIPTION
**Pull request by umamaheswararao**
## What changes were proposed in this pull request?

Fixed the tests to use a predictable way to mark the nodes fail.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-6083

## How was this patch tested?

Fixed the tests to avoid flaky behaviors.

### CODE REVIEW

**Comment by umamaheswararao** [comment]
@fapifta please check this and let's close this JIRA once we commit. Thanks

**Comment by fapifta** [comment]
LGTM, after removing randomness from block allocations, the flakyness of these tests should have gone already, and in the future, if we run again into the condition that caused the flakyness, we will have consistent failures.

Thank you for taking care of this one, I am sorry that I could not get back to this recently after the block allocator changes.
I am closing the other PR.

**Review by fapifta — APPROVED** [review]

**Comment by umamaheswararao** [comment]
Thanks @fapifta for reviewing this. Yeah NP. we need to close this before merge proposals as it states flakiness in tests.

**Pull request merged**
